src/NuGet.Core/NuGet.Common/RuntimeEnvironmentHelper.cs

```diff
@@ -156,5 +158,15 @@ private static bool GetIsLinux()
             return platform == 4;
 #endif
         }
+
+        public static bool IsFreeBSD
```

Comment:
Is there anywhere in NuGet where we will want to do something different on FreeBSD compared to Linux? I think there are places where we do different things on Linux vs Mac, and obviously many where we're different Windows vs everything else. But I don't want to have to find all references to `RuntimeEnvironmentHelper.IsLinux` and add `|| RuntimeEnvironmentHelper.IsFreeBSD`.  If NuGet's code is the same on both, I think it's better to have one property. I'm afraid that I don't really care that FreeBSD and Linux are technically diffrent things if it doesn't affect NuGet's code in any way.

Reply:
> Is there anywhere in NuGet where we will want to do something different on FreeBSD compared to Linux?

Currently not, I think.
So I changed the implementation of `RuntimeEnvironmentHelper.GetIsLinux()` to also return true on FreeBSD if `IS_CORECLR` is defined. (.NET Standard)